data/persian/articles/cej3lzd5e0go.json

```diff
@@ -147,7 +147,7 @@
               {
                 "type": "paragraph",
                 "model": {
-                  "text": "این یک متن متنی است ، که حاوی لینک اینترنتی به صفحه اصلی  است.",
+                  "text": "این یک متن متنی است ، که حاوی لینک اینترنتی به صفحه اصلی است.",
```

Comment:
Confused about the e2e failing for this. Looking at this asset locally, the test looks like it should be failing as the paragraph that rendered wasn't the same as what's in the json file. I then changed it to remove the extra space to see if that satisfied the test and it still can't find it on the page. But when looking for it on the page locally it's there:
![Screenshot 2021-11-01 at 15 54 50](https://user-images.githubusercontent.com/4706195/139701166-cad5445a-57c3-4dd0-8f85-2b505c863a4e.png)

The optimiser might be doing something weird to RTL, but I can't pinpoint it.

Reply:
Removing the extra space in the link values has now seemed to fix the [e2e](https://github.com/bbc/simorgh/runs/4069313776?check_suite_focus=true#step:3:966). Still confused as to why this wasn't failing in the first place as the text on the page on the `latest` branch shows no extra spaces http://localhost:7080/persian/articles/cej3lzd5e0go.amp, but the `json` [file](https://github.com/bbc/simorgh/blob/56d12aadda288fcbd503dcdc46e31ac1c1ab535c/data/persian/articles/cej3lzd5e0go.json#L150) has the extra space